auto-discovery/cloud-aws/main.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/secureCodeBox/secureCodeBox/auto-discovery/cloud-aws/aws"
+	"github.com/secureCodeBox/secureCodeBox/auto-discovery/cloud-aws/config"
+	"github.com/secureCodeBox/secureCodeBox/auto-discovery/cloud-aws/kubernetes"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+func main() {
+	var configFile string
+	flag.StringVar(&configFile, "config", "",
+		"The service will load its initial configuration from this file. "+
+			"Omit this flag to use the default configuration values. "+
+			"Command-line flags override configuration from this file.")
```

Comment:
Straight copy/paste from the kubernetes AutoDiscovery, so not actually true :D

Reply:
I changed the last line so it doesn't tell lies anymore, it now says `Environment variables override some configuration values from this file.`, which seems accurate enough.

I'm also not sure how helpful `Omit this flag to use the default configuration values.` is, because especially for AWS there is no default that makes sense, but it's technically not wrong 🤷